src/main/java/net/praqma/jenkins/configrotator/scm/ConfigRotatorChangeLogParser.java

```diff
@@ -3,10 +3,10 @@
 import hudson.model.AbstractBuild;
 import hudson.scm.ChangeLogParser;
 import hudson.scm.ChangeLogSet;
-import hudson.util.Digester2;
 import org.apache.commons.digester.Digester;
```

Comment:
This has to change to 

```suggestion
import org.apache.commons.digester3.Digester;
```

Reply:
Yikes! thanks for the catch!